### PR TITLE
[GUIDE] How to deploy camino-node on Azure

### PR DESCRIPTION
**Pull request by havan**
This PR adds a new guide page under Camino Node section for deploying camino-node on Microsoft Azure.

Deployed at: https://playground.docs.camino.network/camino-node/azure

### CODE REVIEW

File: docs/camino-node/azure.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+
+- Click **Review and Create**.
+- Finally, select **Create** to deploy your VM.
+
+<figure>
+<img class="zoom" src="/img/azure/azure_vm_19.png#center"/>
+<figcaption align="center"><b>Fig.19</b></figcaption>
+</figure>
+
+- Don't forget to download the public key file to securely connect to your virtual machine in the future.
```
Comment (mo-c4t) [inline docs/camino-node/azure.md]: There is a screenshot here to be added (azure_vm_20.png)
as azure after clicking create it pop up a window to download the private key

File: docs/camino-node/azure.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+### 2. Set Up Virtual Machine Configuration
+
+- Choose the desired **Subscription**.
+- Optionally, create a new resource group specifically for Camino.
+- Input a name for your virtual machine and choose its region.
+
+### 3. Virtual Machine Image and Size
+
+- For the VM image, choose **Ubuntu 22.04** with an x86 architecture.
+- It's recommended to select a machine size of **Standard_D4s_v3**.
```
Comment (mo-c4t) [inline docs/camino-node/azure.md]: This line is shown in Fig 4.

File: docs/camino-node/azure.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+### 3: Install Azure CLI
+
+Install the Azure Command-Line Interface (CLI) to interact with the Azure resources. Find detailed installation instructions [here](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli).
+
+For Ubuntu systems, you can directly install using:
+
+```bash
+curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+```
+
```
Comment (mo-c4t) [inline docs/camino-node/azure.md:61]: Sorry I forgot to add a step here to login 

```bash
az login
```

File: docs/camino-node/azure.md (hunk 1 above)
Comment (havan) [reply docs/camino-node/azure.md]: Fixed

File: docs/camino-node/azure.md (hunk 3 above)
Comment (havan) [reply docs/camino-node/azure.md:61]: Fixed

File: docs/camino-node/azure.md (hunk 2 above)
Comment (havan) [reply docs/camino-node/azure.md]: Fixed

**Review by mo-c4t — APPROVED** [review]

**Pull request merged**